vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -9,6 +9,13 @@ ERR_UA_DETACH=185 {{/* Error to detach UA */}}
 ERR_LINUX_HEADER_INSTALL_TIMEOUT=186 {{/* Timeout to install linux header */}}
 ERR_STRONGSWAN_INSTALL_TIMEOUT=187 {{/* Timeout to install strongswan */}}
 
+ERR_NTP_INSTALL_TIMEOUT=10 {{/*Unable to install NTP */}}
```

Comment:
tool_installs_ubuntu.sh is not part of cse, so no need to use the {{ }} template syntax. We can just add # before the comment

Reply:
should I update the other constants(FIPS related error codes) in tool_installs_ubuntu.sh declared above the ntp/timesyncd/chrony similarly or leave them as it is?